src/pools/D3MAavePool.sol

```diff
@@ -165,7 +165,11 @@ contract D3MAavePool is ID3MPool {
     // Withdraws Dai from Aave in exchange for adai
     // Aave: https://docs.aave.com/developers/v/2.0/the-core-protocol/lendingpool#withdraw
     function withdraw(uint256 wad) external override onlyHub {
+        uint256 prevDai = dai.balanceOf(address(msg.sender));
```

Comment:
Just glancing, but you can probably remove the address casting around msg.sender (it was probably also before).

Reply:
haha, thank you!  artifact of originally typing `address(this)`